ontology/yaml/resources/METERS/entity_types/ABSTRACT.yaml

```diff
@@ -20,8 +20,8 @@ PWM:
   description: "Very basic power and enrgy monitoring only."
   is_abstract: true
   opt_uses:
-  - apparent_power_sensor
-  - reactive_power_sensor
+  - apparentpower_sensor
```

Comment:
Please revert. [This PR](https://github.com/google/digitalbuildings/pull/581) will be removing `apparentpower` and `reactivepower` from the ontology.

Reply:
Noted sir. Thanks